README.md

```diff
@@ -18,40 +18,47 @@ entity2vec computes vector representations of Knowledge Graph entities that pres
+|`feedback_file` | null                   | Path to a DAT file that contains all the couples user-item. If not defined, it assumes that is the file `datasets\<my_dataset>\graphs\feedback.edgelist` |
 
 ## Entity Recommendation
 
-Compute user and item embeddings from a Knowledge Graph encompassing both user feedback information (movielens_1m/graphs/feedback.edgelist) and Linked Open Data information (movielens_1m/graphs/dbpedia_property.edgelist) on the Movielens 1M dataset. It is based on property-specific entity embeddings, which can be already computed or can be computed by calling entity2vec using the command line argument --run_all. It adopts by default the AllItems candidate generation for testing, which means that features are computed for each user-item pair that is not appearing in the training set. Thus, for each user, all items in the database can be ranked to obtain top-N item recommandation.
+Compute user and item embeddings from a Knowledge Graph encompassing both user feedback information (`movielens_1m/graphs/feedback.edgelist`) and Linked Open Data information (`movielens_1m/graphs/dbpedia_property.edgelist`) on the Movielens 1M dataset. It is based on property-specific entity embeddings, which can be already computed or can be computed by calling _entity2vec_ using the command line argument `--run_all`. It adopts by default the _AllItems_ candidate generation for testing, which means that features are computed for each user-item pair that is not appearing in the training set. Thus, for each user, all items in the database can be ranked to obtain top-N item recommendation.
+
+    python src/entity2rec.py --dataset my_dataset --train training_set.dat --test test_set.dat
+
+The command accepts all the params of _entity2vec_ and, in addition:
 
-```
-python src/entity2rec.py --dataset my_dataset --train training_set.dat --test test_set.dat
-```
+|option          | default                |description |
+|----------------|------------------------|------------|
+|`train`         | null **(Required)**    | Path of the train set in DAT format (see below for syntax) |
+|`test`          | null **(Required)**    | Path of the test set in DAT format (see below for syntax)  |
 
-where it is assumed that the training and test set have the format:
 
-```
-user_id item_id rating timestamp
+The training and test set have the format:
 
-```
-if the argument --run_all is provided, entity2vec will be called and generate property specific embeddings and all its command line arguments can be used. Otherwise, the embeddings will be loaded from the emb/ folder.
+    user_id item_id rating timestamp
+
+where the `user_id` should be an integer, eventually preceded by the string `user` (i.e. `13` or `user13`).
+
+If the argument `--run_all` is provided, _entity2vec_ will be called and generate property specific embeddings and all its command line arguments can be used. Otherwise, the embeddings will be loaded from the `emb/` folder.
```

Comment:
Are you sure? I think that in this line, it means that e2r calls e2v. It is not like this?